frc_characterization/logger_analyzer/data_analyzer.py

```diff
@@ -1477,6 +1003,12 @@ def compute_accel(self, data, window):
 
         return dat
 
+    def is_valid(self, a_tuple):
+        for a in a_tuple:
+            if a is None:
+                return False
+        return True
```

Comment:
Apparently numpy dislikes using None in a tuple of np arrays so I had to make an is_valid method that iterates through the tuple to make sure none of the elements are null.

Reply:
Is this not the `all()` function?

Reply:
No, np arrays have an ambiguous definition for True or False (a.any() or a.all()) so `all()` leads to a Value Error.

Reply:
Huh?

<s>From what I can see you call `is_valid` like so: [1]
```py
self.is_valid((sf_l, sb_l, ff_l, fb_l, sf_r, sb_r, ff_r, fb_r)),
```
Thus, your code iterate through all of these and checks if they are `== None`. 
</s>

oh, ic. `__eq__` doesn't actually return a boolean. very weird

as a sidenote, the function could be
```py
def is_valid(self, *a_tuple):
    for a in a_tuple:
        if a is None:
            return False
    return True
```

and then called like:

```py
self.is_valid(sf_l, sb_l, ff_l, fb_l, sf_r, sb_r, ff_r, fb_r),
```

Reply:
Ok, I'll add that in.